packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -168,12 +177,18 @@ export class WaferMap extends FoundationElement {
         this.canvasContext = this.canvas.getContext('2d', {
             willReadFrequently: true
         })!;
+        this.hoverHandler.connect();
+        this.experimentalHoverHandler.connect();
```

Comment:
Isn't this problematic in that both the `hoverHandler` and `experimentalHoverHandler` are now responding to the same event?